phpunit/class-wp-rest-url-details-controller-test.php

```diff
@@ -788,19 +739,19 @@ public function data_get_description() {
 			// Happy paths with HTML tags or entities in the description.
 			'with HTML tags'                             => array(
 				'<meta name="description" content="<strong>Description</strong>: has <em>HTML</em> tags">',
-				'<strong>Description</strong>: has <em>HTML</em> tags',
+				'Description: has HTML tags',
 			),
 			'with content first and HTML tags'           => array(
 				'<meta content="<strong>Description</strong>: has <em>HTML</em> tags" name="description">',
-				'<strong>Description</strong>: has <em>HTML</em> tags',
+				'Description: has HTML tags',
 			),
 			'with HTML tags and other attributes'        => array(
 				'<meta first="first" name="description" third="third" content="<strong>Description</strong>: has <em>HTML</em> tags" fifth="fifth>',
-				'<strong>Description</strong>: has <em>HTML</em> tags',
+				'Description: has HTML tags',
 			),
 			'with HTML entities'                         => array(
 				'<meta name="description" content="The &lt;strong&gt;description&lt;/strong&gt; meta &amp; its attribute value"',
-				'The <strong>description</strong> meta & its attribute value',
+				'The description meta & its attribute value',
```

Comment:
@getdave Notice both new behaviors:

- HTML is stripped
- non-HTML entity (i.e. ` &amp;`) decoded to `&`